### PR TITLE
Timelock and OTC docs

### PR DESCRIPTION
**Pull request by SwayStar123**
## Type of change

- Documentation

## Changes

Adds or updates documentation to follow src-2 standard

### CODE REVIEW

File: OTC-swap-predicate/project/predicates/swap-predicate/src/main.sw [hunk 1]
```diff
@@ -16,12 +16,23 @@ use std::{
...
+    /// The amount of the ask.
     ASK_AMOUNT: u64 = 42,
+    /// The token to be paid.
     ASK_TOKEN: ContractId = ContractId::from(0x0101010101010101010101010101010101010101010101010101010101010101),
+    /// The receiver of the ask.
     RECEIVER: Address = Address::from(0x09c0b2d1a486c439a87bcba6b46a7a1a23f3897cc83a94521a96da5c23bc58db),
 }
 
-/// Order / OTC swap Predicate
+/// Order / OTC swap Predicate. Like a limit order.
```
Comment (Braqzen) [inline OTC-swap-predicate/project/predicates/swap-predicate/src/main.sw:27]:
```suggestion
/// Validates conditions within the transaction to perform a swap
```

File: OTC-swap-predicate/project/predicates/swap-predicate/src/main.sw [hunk 2]
```diff
@@ -16,12 +16,23 @@ use std::{
...
+///
+/// # Additional Information
+///
+/// The user can cancel their own order by spending the output coin to themselves.
```
Comment (Braqzen) [inline OTC-swap-predicate/project/predicates/swap-predicate/src/main.sw:31]:
```suggestion
/// The user can cancel their order by including an input coin from themselves.
```

File: OTC-swap-predicate/project/predicates/swap-predicate/src/main.sw [hunk 3]
```diff
@@ -16,12 +16,23 @@ use std::{
 };
 
 configurable {
+    /// The amount of the ask.
     ASK_AMOUNT: u64 = 42,
+    /// The token to be paid.
     ASK_TOKEN: ContractId = ContractId::from(0x0101010101010101010101010101010101010101010101010101010101010101),
+    /// The receiver of the ask.
```
Comment (Braqzen) [inline OTC-swap-predicate/project/predicates/swap-predicate/src/main.sw:23]:
```suggestion
    /// The receiver to whom the swapped asset will be sent.
```

**Review by Braqzen — COMMENTED** [review]
2 Apps = 2 PRs next time

File: timelock/project/contracts/timelock-contract/src/interface.sw [hunk 4]
```diff
@@ -4,81 +4,93 @@ use ::data_structures::{Asset, ExecutionRange};
...
-    /// - `asset_id`: The identifier of an asset
+    /// * `asset_id`: [ContractId] - The identifier of an asset.
+    ///
+    /// # Returns
+    ///
+    /// * [u64] - The amount of `asset_id` in the contract.
     fn balance(asset_id: ContractId) -> u64;
 
-    /// Returns the (MINIMUM_DELAY, MAXIMUM_DELAY) values
+    /// Returns the (MINIMUM_DELAY, MAXIMUM_DELAY) values.
```
Comment (bitzoic) [inline timelock/project/contracts/timelock-contract/src/interface.sw:68]:
```suggestion
    /// Returns the delay values.
    ///
    /// # Returns
    ///
    /// * [u64] - The minimum delay value.
    /// * [u64] - The maximum delay value. 
```
Double-check that these are correct

**Comment by Braqzen** [comment]
@SwayStar123 add a format command to your commits (pre-commit command).

**Review by Braqzen — APPROVED** [review]

**Pull request merged**